pytorch_lightning/accelerators/accelerator.py

```diff
@@ -401,20 +401,6 @@ def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: _PATH) -> None:
         """
         self.training_type_plugin.save_checkpoint(checkpoint, filepath)
 
-    @property
-    def call_configure_sharded_model_hook(self) -> bool:
```

Comment:
consider deprecation as it used to be public API
on the other hand, Accelerators are meant to be for internal use only...
cc: @awaelchli

Reply:
that's ok imo. the hook is not really valuable as a way to extend the plugins and it really only was a hack for the sharded integration :)